.github/workflows/test.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: [19.x, 18.x, 16.x, 14.x]
+        version: [20.x, 19.x, 18.x, 16.x, 14.x]
```

Comment:
Maybe we should drop 19.x since it is now neither current nor LTS?

Reply:
Good idea. I just removed 19.x in the latest push.